doc/z-chap03.xml

```diff
@@ -33,70 +35,107 @@
+      </Alt>,
+      &i;<M>\in</M>&bfn;, &j;<M>\in</M> -&bfn; and
+      <Alt Not = "Text">
         <Display>(i,-k(1))\in x,\ (k(1),k(2))\in y,\ (-k(2),-k(3))\in x,\ \ldots,\
-          (-k(2s-2),-k(2s-1))\in x,\ (k(2s-1),-j)\in y</Display>
+          (-k(2s-2),-k(2s-1))\in x,\ (k(2s-1),j)\in y;</Display>
       </Alt>
       <Alt Only = "Text">
-        <C>r=2s-1</C> for some <C>s</C><M>\geq</M><C>1</C> and<P/>
         <C>(i,-k(1))</C><M>\in</M><C>x</C>, <C>(k(1),k(2))</C><M>\in</M><C>y</C>,
         <C>(-k(2),-k(3))</C><M>\in</M><C>x</C>, <M>\ldots</M>,
         <C>(-k(2s-2),-k(2s-1))</C><M>\in</M><C>x</C>,
-        <C>(k(2s-1),-j)</C><M>\in</M><C>y</C>
+        <C>(k(2s-1),j)</C><M>\in</M><C>y</C>;
       </Alt>
     </Item>
- 
     <Item>
       <Alt Not = "Text">
-        <M>r=2s</M> for some <M>s\geq 1</M> and either:
+        <M>r=2s</M> for some <M>s\geq 1</M>
+      </Alt>
+      <Alt Only = "Text">
+        <C>r=2s</C> for some <C>s</C><M>\geq</M><C>1</C>
+      </Alt>,
+      and either
+      &i;,&j;<M>\in</M>&bfn;, and
+      <Alt Not = "Text">
         <Display>(i,-k(1))\in x,\ (k(1),k(2))\in y,\ (-k(2),-k(3))\in x,\ \ldots,
-          (k(2s-1), k(2s))\in y,\ (-k(2s), j)\in x</Display>
-        or
-        <Display>(-i,k(1))\in y,\ (-k(1),-k(2))\in x,\ (k(2),k(3))\in y,\ \ldots,
-          (-k(2s-1), -k(2s))\in x,\ (k(2s), -j)\in y.</Display>
+          (k(2s-1), k(2s))\in y,\ (-k(2s), j)\in x,</Display>
       </Alt>
       <Alt Only = "Text">
-        <C>r=2s</C> for some <C>s</C><M>\geq</M><C>1</C> and either:
         <P/>
         <C>(i,-k(1))</C><M>\in</M><C>x</C>,
         <C>(k(1),k(2))</C><M>\in</M><C>y</C>,
         <C>(-k(2),-k(3))</C><M>\in</M><C>x</C>,
         <M>\ldots</M>,
         <C>(k(2s-1),k(2s))</C><M>\in</M><C>x</C>,
-        <C>(-k(2s),j)</C><M>\in</M><C>y</C><P/>
-        or<P/>
-        <C>(-i,k(1))</C><M>\in</M><C>y</C>,
+        <C>(-k(2s),j)</C><M>\in</M><C>y</C>,<P/>
+      </Alt>
+      or &i;,&j;<M>\in</M>-&bfn;, and
+      <Alt Not = "Text">
+        <Display>(i,k(1))\in y,\ (-k(1),-k(2))\in x,\ (k(2),k(3))\in y,\ \ldots,
+          (-k(2s-1), -k(2s))\in x,\ (k(2s), j)\in y.</Display>
+      </Alt>
+      <Alt Only = "Text">
+        <P/>
+        <C>(i,k(1))</C><M>\in</M><C>y</C>,
         <C>(-k(1),-k(2))</C><M>\in</M><C>x</C>,
         <C>(k(2),k(3))</C><M>\in</M><C>y</C>,
         <M>\ldots</M>,
         <C>(-k(2s-1),-k(2s))</C><M>\in</M><C>y</C>,
-        <C>(k(2s),-j)</C><M>\in</M><C>x</C>.<P/>
+        <C>(k(2s),j)</C><M>\in</M><C>x</C>.<P/>
       </Alt>
     </Item>
- 
   </List>
 
-  This product can be shown to be associative, and so the collection of
-  bipartitions of any particular degree is a monoid; the identity element is the
-  partition
-  <Alt Not = "Text"><M>\left\{\{i,-i\}:i\in\mathbf{n}\right\}</M>.</Alt>
-  <Alt Only = "Text">{{<C>i</C>,-<C>i</C>}:<C>i</C><M>\in</M><E>n</E>}.</Alt>
-  A bipartition is a unit if and only if each block is of the form
```

Comment:
I believe each `<C>i</C>` in the following paragraph should be replaced by `<M>i</M>` (and the same for `-i`).